src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -186,11 +195,57 @@ private void linkToCase(User defendantUser, User anonymousCaseWorker, String let
             letterHolderId
         );
 
-        coreCaseDataService.update(
+        String defendantEmail = defendantUser.getUserDetails().getEmail();
+        CaseDetails caseDetails = coreCaseDataService.update(
             defendantUser.getAuthorisation(),
-            CCDCase.builder().id(Long.valueOf(caseId)).defendantId(defendantId).build(),
+            CCDCase.builder()
+                .id(Long.valueOf(caseId))
+                .defendantId(defendantId)
+                .defendantEmail(defendantEmail)
+                .build(),
             CaseEvent.LINK_DEFENDANT
         );
+
+        scheduleEmailNotificationsForDefendantResponse(caseId, defendantId, defendantEmail, caseDetails);
+    }
+
+    private void scheduleEmailNotificationsForDefendantResponse(
+        String caseId,
+        String defendantId,
+        String defendantEmail,
+        CaseDetails caseDetails
+    ) {
+        Claim claim = ccdCaseDataToClaim.to(caseDetails.getId(), caseDetails.getData());
+        LocalDate responseDeadline = claim.getResponseDeadline();
+        String defendantName = claim.getClaimData().getDefendant().getName();
+        String claimantName = claim.getClaimData().getClaimant().getName();
+        ImmutableMap.Builder<String, Object> emailData = ImmutableMap.builder();
+        emailData.put("caseId", caseId);
+        emailData.put("caseReference", claim.getReferenceNumber());
+        emailData.put("defendantEmail", defendantEmail);
+        emailData.put("defendantId", defendantId);
+        emailData.put("defendantName", defendantName);
+        emailData.put("claimantName", claimantName);
+        emailData.put("responseDeadline", responseDeadline);
+
+        jobService.scheduleJob(
+            JobData.builder()
+                .id(UUID.randomUUID().toString())
```

Comment:
What about including some identifiable details in ID to make trouble shooting easier?

For example we could have `reminder:defence-due-in-5-days:000MC001-UUID.randomUUID().toString()` and `reminder:defence-due-in-1-day:000MC001-UUID.randomUUID().toString()`